roles/kubernetes-apps/external_provisioner/local_path_provisioner/defaults/main.yml

```diff
@@ -6,3 +6,4 @@ local_path_provisioner_reclaim_policy: Delete
 local_path_provisioner_claim_root: /opt/local-path-provisioner/
 local_path_provisioner_is_default_storageclass: "true"
 local_path_provisioner_debug: false
+local_path_provisioner_image_pull_policy: "Always"
```

Comment:
see roles/kubespray-defaults/defaults/main.yaml
```
k8s_image_pull_policy: IfNotPresent
```
Most deployers want to use just one variable to control this globally. Please just use that var and not add another.

Reply:
Thanks, I've changed the file to use the globally controlled variable and removed the additional var I added.